### ISSUE TITLE
interface.stat test failure

### ISSUE THREAD

**Issue by danielmewes** [rethinkdb/rethinkdb#5035]
```py
Traceback (most recent call last):
  File "/home/ssd3/daniel/rethinkdb/test/interface/stat.py", line 249, in <module>
    compare_global_and_individual_stats(all_stats, also_stats)
  File "/home/ssd3/daniel/rethinkdb/test/interface/stat.py", line 193, in compare_global_and_individual_stats
    assert a['query_engine']['read_docs_per_sec'] > 0
AssertionError
Command '$RETHINKDB/test/interface/stat.py' returned non-zero exit status 1
```

**Comment by danielmewes** [comment]
Turns out this failure is timing dependent. Which makes sense, since it's failing on a `_per_sec` stat.

**Comment by danielmewes** [comment]
Fix in review 3625

**Comment by danielmewes** [comment]
Fixed in v2.3.x and next via 7407f38 and 6f1ad29 respectively.
